var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -224,11 +234,17 @@ class Ascent(CMakePackage, CudaPackage):
         depends_on("vtk-m~openmp", when="@0.9.0: ~openmp")
         depends_on("vtk-m~cuda", when="@0.9.0: ~cuda")
         depends_on("vtk-m+cuda", when="@0.9.0: +cuda")
+        depends_on("vtk-m~rocm", when="@0.9.0: ~rocm")
+
+        depends_on("kokkos +rocm", when="+vtkh +rocm")
+        depends_on("vtk-m+kokkos+rocm", when="@0.9.0: +rocm")
         depends_on("vtk-m+fpic", when="@0.8.0:")
         depends_on("vtk-m~shared+fpic", when="@0.8.0: ~shared")
-        # Ascent defaults to C++11
-        depends_on("kokkos cxxstd=11", when="+vtkh ^vtk-m +kokkos")
-        depends_on("kokkos@3.7.02", when="@0.9.3: +vtkh ^vtk-m +kokkos")
+        with when("^vtk-m +kokkos"):
+            depends_on("kokkos@3.7.02", when="@0.9.3:")
+            # Ascent defaults to C++11
+            depends_on("kokkos cxxstd=11", when="~rocm")
```

Comment:
Instead of this can we extract the cxxstd from kokkos and pass that to ascent? Ascent doesn't actually care about the standard in the public API, so I think it is fine to infer it.